CONTRIBUTING.md

```diff
@@ -101,4 +105,4 @@ After each commit TravisCI will check your new code against the formatting guide
 
 We will only merge PRs that pass the TravisCI checks.
 
-If your check fails don't worry you will still be able to make changes and make your code pass the checks.
\ No newline at end of file
+If your check fails don't worry you will still be able to make changes and make your code pass the checks.
```

Comment:
Why is this line listed as a change?

Reply:
I can't see it when I run git diff. It already existed in the document. Maybe some spaces/enters affected it?